controller/array_action/array_mediator_interface.py

```diff
@@ -243,12 +243,12 @@ def get_object_by_id(self, object_id, object_type):
         raise NotImplementedError
 
     @abstractmethod
-    def create_snapshot(self, name, volume_name, pool_id=None):
+    def create_snapshot(self, name, volume_id, pool_id):
```

Comment:
shouldn't `pool_id` be just `pool` now?

Reply:
why?

Reply:
because non-ds8k mediators use pool name

Reply:
even before this PR - if `api_volume.pool` is an id (and not a name), then same goes for the `pool_name` member name of `Volume`

Reply:
done

Reply:
I still see `pool_name` in `class Volume` - is it always a name?

Reply:
no, changed to `pool`